setup.cfg

```diff
@@ -61,6 +61,7 @@ install_requires =
     jax_dataclasses >= 1.4.0
     pptree
     rod >= 0.2.0
+    trimesh
```

Comment:
Can you please add this new dependency in the environment.yml file for conda-based installations?

Reply:
This has not yet been resoved